### ISSUE TITLE
Missing "angle" in Position

### ISSUE THREAD

**Issue by lnx85** [DeebotUniverse/client.py#491]
### Checks

- [X] I have searched the existing issues and no issue is describing my issue
- [X] I have checked the [FAQ](https://deebot.readthedocs.io/home/faq/)
- [X] I have checked the [documentation](https://deebot.readthedocs.io/integrations/home-assistant)
- [X] I have installed the latest version

### The problem

The Position object is missing informations about the robot angle.

### On which deebot device (vacuum) you have the issue?

Deebot N8

### Which version of the deebot-client are you using?

7.2.0

### Country

it

### Continent

eu

### Anything in the logs that might be useful for us?

```txt
Log for PositionsEvent (after a GetPos command).

Notify subscribers with PositionsEvent(positions=[Position(type=<PositionType.DEEBOT: 'deebotPos'>, x=-396, y=10), Position(type=<PositionType.CHARGER: 'chargePos'>, x=-396, y=10)])

The same request made with ecovacs-deebot.js (https://github.com/mrbungle64/ecovacs-deebot.js/blob/master/example/mapInfos.js) returns and object containg "x", "y" and "a" where "a" is bot orientation in degrees.
```

### Additional information

The GetPos command is not documented but available in source code.

**Comment by edenhaus** [comment]
What would be a use-case for the angle?

**Comment by lnx85** [comment]
I was using that parameter to show the direction of the robot while moving. ![image](https://github.com/DeebotUniverse/client.py/assets/1611929/10bf7804-7889-4c7e-96bf-33b406d300b4)

**Comment by edenhaus** [comment]
Please feel free to add the angle via a PR :)

As I like your floorplan, can you share it?

**Comment by lnx85** [comment]
> Please feel free to add the angle via a PR :)
> 
> As I like your floorplan, can you share it?

Map has been created with Wondershare eDraw Max. In Home Assistant I have an automation that "translate" bot coordinates (x, y) to a float from 0 to 1 (used to position the Bot marker on the map with CSS left and top in percentage).

Angle (a) is used to rotate the bot image (0deg to 360deg)

**Comment by lnx85** [comment]
Implemented in #499.

**Comment by lnx85** [comment]
Implemented in #499.
